core/utils.js

```diff
@@ -465,18 +464,18 @@ Blockly.utils.replaceMessageReferences = function(message) {
  *     Otherwise, false.
  */
 Blockly.utils.checkMessageReferences = function(message) {
-  var isValid = true; // True until a bad reference is found
+  var isValid = true;  // True until a bad reference is found.
 
   var regex = /%{BKY_([a-zA-Z][a-zA-Z0-9_]*)}/g;
   var match = regex.exec(message);
-  while (match != null) {
+  while (match) {
     var msgKey = match[1];
-    if (Blockly.utils.getMessageArray_()[msgKey] == null) {
```

Comment:
It makes no difference since `null == undefined`, but the value when no message matches is `undefined`, not `null`.